tests/helpers/MockToolRegistry.ts

```
@@ -203,10 +296,10 @@ export class MockToolRegistry {
               throw new AgentNotInRoomError(args.agentName, args.roomName);
             }
             
-            const messages = this.dataLayer.getMessages(args.roomName, args.limit, args.before);
+            const messages = this.dataLayer.getMessages(args.roomName, args.limit, args.before, args.offset);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)